index.html

```diff
@@ -34,6 +35,179 @@ <h1 class="header-text">I’m Hamma
                 <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
             </ul>
         </section>
+        <section class="work-cards-container">
+            <div class="work-card work-card1">
+                <img src="img/works/work1.png" alt="work1">
+
+                <h2 class="work-title">Tonik</h2>
+                <div class="work-info">
+                    <p class="first">CANOPY</p>
+                    <p class="second">• Back End Dev</p>
+                    <p class="second">• 2015</p>
+                </div>
+                <div class="word-desc">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-lang">
+                    <ul>
+                        <li>
+                            <p class="lang-text">html</p>
+                        </li>
+                        <li>
+                            <p class="lang-text">css</p>
+                        </li>
+                        <li>
+                            <p class="lang-text">javascript</p>
+                        </li>
+                    </ul>
+                </div>
+                <button type="button" class="work-btn btn">
+                        See Project
+                    </button>
+            </div>
+            <div class="work-card work-card2">
+                <img src="img/works/work2.png" alt="work1">
+
+                <h2 class="work-title">Multi-Post
```

Comment:
- [ ] please use the correct figma font size and font weight for your card titles